lib/crypto.js

```diff
@@ -205,12 +101,12 @@ module.exports.encrypt = function (message/* : Uint8Array */,
 }
 
 /**
- * Decrypts and verifies a message using Nacl secretbox. Returns false if
+ * Decrypts and verifies a message using Nacl secretbox. Returns null if
  * verification fails.
  * @param {Uint8Array} ciphertext
  * @param {Uint8Array} nonce
  * @param {Uint8Array} secretboxKey
- * @returns {Uint8Array|boolean}
+ * @returns {Uint8Array?}
```

Comment:
this is a change from tweetnacl 1.0.0 - see https://github.com/dchest/tweetnacl-js/issues/132. AFAIK it only affected tests (which were fixed)